cypress/e2e/models/administration/credentials/credentialsJira.ts

```diff
@@ -0,0 +1,107 @@
+import { Credentials } from "./credentials";
+import { CredentialsJiraData } from "../../../types/types";
+import {
+    cancelForm,
+    click,
+    clickByText,
+    exists,
+    inputText,
+    notExists,
+    submitForm,
+    validateValue,
+} from "../../../../utils/utils";
+import { passwordInput, usernameInput } from "../../../views/credentials.view";
+import { button, CredentialType } from "../../../types/constants";
+
+export class CredentialsJira extends Credentials {
+    type = CredentialType.jira;
+    email: string;
+    token: string;
+
+    constructor(credentialsJiraData: CredentialsJiraData) {
+        super();
+        this.init(credentialsJiraData);
+    }
+
+    protected init(credentialsJiraData: CredentialsJiraData) {
+        const { name, description, email, token } = credentialsJiraData;
+        this.name = name;
+        this.description = description;
+        this.email = email;
+        this.token = token;
+    }
+
+    protected fillEmail() {
+        inputText(usernameInput, this.email);
+    }
+
+    protected fillToken() {
+        inputText(passwordInput, this.token);
+    }
+
+    protected selectCredType() {
```

Comment:
**question**: What is the purpose of this method? It's protected but isn't used inside the class. And in the Jira credentials form there is no "Username/Password" text as far as I see. I guess it can be safely deleted.

Reply:
Not relevant any more